gradle-baseline-java-config/resources/checkstyle/checkstyle.xml

```diff
@@ -53,6 +53,9 @@
         <property name="optional" value="true"/>
     </module>
     <module name="SuppressWarningsFilter"/> <!-- baseline-gradle: README.md -->
+    <module name="BeforeExecutionExclusionFileFilter">
+        <property name="fileNamePattern" value="module\-info\.java$"/>
+    </module>
```

Comment:
As seen in https://checkstyle.sourceforge.io/config_filefilters.html#BeforeExecutionExclusionFileFilter